sdk/cognitivelanguage/Azure.AI.Language.Conversations/api/Azure.AI.Language.Conversations.netstandard2.0.cs

```diff
@@ -1,49 +1,27 @@
 namespace Azure.AI.Language.Conversations
 {
+    public partial class AnalysisParameters
+    {
+        public AnalysisParameters() { }
+        public string ApiVersion { get { throw null; } set { } }
+    }
     public partial class AnalyzeConversationOptions
     {
         public AnalyzeConversationOptions(string projectName, string deploymentName, string query) { }
         public string DeploymentName { get { throw null; } }
         public string DirectTarget { get { throw null; } set { } }
         public bool? IsLoggingEnabled { get { throw null; } set { } }
         public string Language { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, Azure.AI.Language.Conversations.Models.AnalysisParameters> Parameters { get { throw null; } }
+        public System.Collections.Generic.IDictionary<string, Azure.AI.Language.Conversations.AnalysisParameters> Parameters { get { throw null; } }
```

Comment:
What do these do? The name might raise some questions.

Can discuss offline and "fix" in another PR.